framework/decode/vulkan_replay_dump_resources_compute_ray_tracing.cpp

```diff
@@ -873,7 +867,23 @@ VkResult DispatchTraceRaysDumpingContext::DumpDispatchTraceRays(
             }
         }
 
-        GenerateOutputJsonDispatchInfo(qs_index, bcb_index, disp_index, i);
+        VulkanDumpDrawCallInfo draw_call_info{};
+        draw_call_info.type                         = DumpResourceType::kDispatchInfo;
+        draw_call_info.instance_table               = instance_table;
+        draw_call_info.device_table                 = device_table;
+        draw_call_info.object_info_table            = &object_info_table;
+        draw_call_info.device_info                  = device_info;
+        draw_call_info.original_command_buffer_info = original_command_buffer_info;
+        draw_call_info.bcb_index                    = bcb_index;
+        draw_call_info.cmd_buf_index                = i;
```

Comment:
Here you need `draw_call_info.cmd_index = disp_index;` instead of `draw_call_info.cmd_buf_index = i;`

Reply:
So basically `i` is not really needed in this loop. I would suggest converting the for loop from
```
    for (size_t i = 0; i < dispatch_indices.size(); ++i)
    {
        const uint64_t disp_index = dispatch_indices[i];
```
into
```
    for (const uint64_t disp_index :  dispatch_indices)
    {
```